src/main/java/calculator/Tokens.java

```diff
@@ -0,0 +1,44 @@
+package calculator;
+
+public class Tokens {
+    private final String[] values;
+
+    public Tokens(String expression) {
+        validate(expression);
+        this.values = parse(expression);
+    }
```

Comment:
```suggestion
    public Tokens(String expression) {
        this(parse(expression));
    }
    
    public Tokens(String[] values) {
        this.values = values;
    }
```
주 생성자를 추가해 필드를 초기화하는 습관을 들여보면 어떨까?